packages/ia-topnav/src/ia-topnav.js

```diff
@@ -220,13 +222,13 @@ export default class IATopNav extends LitElement {
    */
   get userMenuItems() {
     const basicItems = this.menus.user;
-    
+
     let adminItems = this.menus.userAdmin;
     if (this.canManageFlags) {
       adminItems = adminItems.concat(this.menus.userAdminFlags);
     }
 
-    return this.admin
+    return this.admin && !this.suppressAdminLinks
```

Comment:
Q: Instead of adding new state for suppression, would it work to just check whether `this.itemIdentifier` is set? I believe that property should be undefined on non-item/non-collection pages already.

Reply:
Addressed your suggestion

Reply:
Applied the above suggestion and working great!